ClientTests/FavoritesViewControllerTests.swift

```diff
@@ -0,0 +1,145 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import XCTest
+import Data
```

Comment:
Make it `@testable` and tests should pass

Reply:
Thanks, fixed!